.github/workflows/tests.yml

```diff
@@ -0,0 +1,44 @@
+name: Go Tests
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 11 * * *"
```

Comment:
Can you change the time to not conflict with other tests as we are sharing the same env? Flask is running at 11, Fast API at 10.

Reply:
Okay updated it to run at 12.